excretor/src/env.rs

```diff
@@ -10,6 +10,14 @@ pub struct EnvVars {
     pub tummy_password: String,
     #[arg(env)]
     pub tummy_port: String,
+    #[arg(env)]
+    pub slack_client_id: String,
+    #[arg(env)]
+    pub slack_client_secret: String,
+    #[arg(env)]
+    pub slack_signing_secret: String,
+    #[arg(env, default_value = "")]
```

Comment:
Isn't this a required field?

Reply:
yeah, was trying to implement some optional feature, forgot to revert, mb